### PR TITLE
Alfoa/relap5 defect

### LINKED ISSUE #1143: [DEFECT] Relap5 Interface Minor Edit mis-collection
--------
Defect Description
--------

**Describe the defect**
The user reported a mis-collection of the results from RELAP5 in case of a warning injected
by the RELAP5 code at the end of the minor edits.

##### What did you expect to see happen?
The RELAP5 interface to work.

##### What did you see instead?
A failure of the interface in collection data

##### Do you have a suggested fix for the development team?
Fix the parser.

**Describe how to Reproduce**
1.  Introduce in the output file of RELAP5 the following strings at the end of a MINOR EDIT block:
`0Reflood calculation became suspect at timehy = 2.52388E+04 because the code did not allocate enough storage for the axial
nodes required for this transient; storage can not be increased by input data or jcl change, but the number of axial nodes can.
0 nmy =       161 iglrfl(ih) =       159 numbers of axial    nodes are ... 
           2           2           4          16           8           8           8           8
           8           8           8           8           8           8           8           8
           8           8           8           8           8
 Reducing time step due to max number of repeats                584470
 Reducing time step due to max number of repeats                588786`

**Screenshots and Input Files**
N/A

**Platform (please complete the following information):**
 - OS: [e.g. iOS] Any
 - Version: [e.g. 22] Latest
 - Dependencies Installation: [CONDA or PIP] CONDA

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [x] 1. Is it tagged with a type: defect or task?
- [x] 2. Is it tagged with a priority: critical, normal or minor?
- [x] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [x] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [x] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. If the issue is a defect, is the defect fixed?
- [x] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] 4. If the issue is a defect, does it impact the latest release branch? If yes, is there any issue tagged with release (create if needed)?
- [x] 5. If the issue is being closed without a pull request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by alfoa**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
Closes #1143

##### What are the significant changes in functionality due to this change request?
Modified the interface of RELAP5 to correctlty collect the results if Warnings are issues.
In addition, a test case is modified in order to capture the fix.

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?
- [x] 9. If any test used as a basis for documentation examples (currently found in `raven/tests/framework/user_guide` and `raven/docs/workshop`) have been changed, the associated documentation must be reviewed and assured the text matches the example.

### CODE REVIEW

**Comment by wangcj05** [comment]
The changes in PR #1140 is also shown in the PR. @alfoa

**Comment by alfoa** [comment]
yes I rebased it.
Andrea

File: framework/CodeInterfaces/RELAP5/relapdata.py [hunk 1]
```diff
@@ -104,53 +104,44 @@ def readMinorBlock(self,lines,i):
...
-          flagg2=1
+    while(keepReading):
+      tempkeys=[]
+      temp1 = re.split('\\s{2,}|\n',lines[i])[:-1]
+      temp2 = [lines[i+1][j:j+13].strip() for j in range(0, len(lines[i+1]), 13)][:-1]
+      temp2 = ['_'.join(key.split()) for key in temp2]
+      tempArray=[]
+      for j in range(len(temp1)):
+        tempkeys.append(temp1[j]+'_'+temp2[j])
+        edit_keys.append(temp1[j]+'_'+temp2[j])
```
Comment (wangcj05) [inline framework/CodeInterfaces/RELAP5/relapdata.py]: edit_keys --> editKeys?

Comment (alfoa) [reply framework/CodeInterfaces/RELAP5/relapdata.py]: Actually I remove it since it is not used anymore

**Comment by wangcj05** [comment]
Changes are good to me. Please fix the failed tests. @alfoa

**Comment by alfoa** [comment]
@wangcj05 can you merge it?

**Comment by wangcj05** [comment]
I have gone through the recent changes, and they are all good.

**Comment by wangcj05** [comment]
Checklist is good, and this PR can be merged.

**Comment by wangcj05** [comment]
@alfoa This PR has been merged, please send an email to the user regarding this fix.

**Pull request merged**